Lib/test/test_logging.py

```diff
@@ -4406,6 +4406,29 @@ def test_basic(self):
         self.assertTrue(found, msg=msg)
 
 
+class CachingTest(BaseTest):
+    def test_caching(self):
+        root = self.root_logger
+        level1 = logging.getLogger("abc")
```

Comment:
These are loggers, not levels, and the names should reflect that. I understand you're talking about level in the logger hierarchy, but still, the use of `level` for the loggers is potentially confusing.